aderyn_core/src/detect/experimental/sibling.rs

```diff
@@ -78,9 +78,10 @@ mod sibling_demo_tests {
     }
 
     #[test]
+    #[serial(fc_solc)]
```

Comment:
Tests with key `fc_solc` wrapped in `#[serial()]` will happen in sequential order. This was done to avoid the solc binary poisoned/busy error

Reply:
More context about error here https://github.com/Cyfrin/aderyn/pull/333#issuecomment-2068039301